cleanlab/util.py

```diff
@@ -330,11 +327,11 @@ def confusion_matrix(true, pred):
     ----------
     y : np.array 1d
       Contains labels.
-      Assumes s and y contains the same distinct set of labels.
+      Assumes labels and y contains the same distinct set of labels.
 
-    s : np.array 1d
-      Contains labels.
-      Assumes s and y contains the same distinct set of labels.
+    labels : np.array 1d
```

Comment:
L316: parameter and doctoring mismatch: (true, pred) -> (y, labels)
L319: typo: represtented

Reply:
nice catch. thanks